go/registry/api/sanity_check.go

```diff
@@ -33,6 +39,14 @@ func (g *Genesis) SanityCheck(baseEpoch epochtime.EpochTime) error {
 		return err
 	}
 
+	// Check that all entities have enough stake to be registered.
+	qZero := quantity.NewQuantity()
+	if entityThreshold := stakeThresholds[staking.KindEntity]; entityThreshold.Cmp(qZero) > 0 {
```

Comment:
in other places, we use `!qqqq.IsZero()`

Reply:
Aha, ok. I will update the code to use that form.

Reply:
Done.